test/e2e/sched/sched_test.go

```diff
@@ -174,5 +181,60 @@ var _ = Describe("[Scheduler] imageReplacement", func() {
 				return true
 			}).WithTimeout(time.Minute * 2).WithPolling(time.Second * 30).Should(BeTrue())
 		})
+		It("should reflect changes in cacheResyncPeriod when configured", func() {
+			nroSchedObj := objects.TestNROScheduler()
+			Expect(e2eclient.Client.Get(context.TODO(), client.ObjectKeyFromObject(nroSchedObj), nroSchedObj)).ToNot(HaveOccurred())
+
+			t := time.Minute * 5
+			nroSchedObj.Spec.CacheResyncPeriod = &metav1.Duration{Duration: t}
+			Expect(e2eclient.Client.Update(context.TODO(), nroSchedObj)).ToNot(HaveOccurred())
+
+			By("checking cacheResyncPeriod under the CR's Status")
+			Expect(e2eclient.Client.Get(context.TODO(), client.ObjectKeyFromObject(nroSchedObj), nroSchedObj)).ToNot(HaveOccurred())
+			Expect(nroSchedObj.Status.CacheResyncPeriod.Duration).To(Equal(nroSchedObj.Spec.CacheResyncPeriod.Duration))
```

Comment:
should this be a `Eventually` loop (which we expect to converge in 1 or 2 iterations, but still)